progress/humaans.go

```diff
@@ -0,0 +1,25 @@
+package progress
```

Comment:
This is actually a play on words, originating from the ferengis in star trek, referring to the silly things humans do.